docs/docs/docs/getting-started/installation.md

```
@@ -295,6 +295,20 @@ devcontainer build --workspace-folder .
 devcontainer up --workspace-folder .
 ```
 
+#### Import Sample Data
+
+1. Once the server is running, open a new terminal session.
+2. Open a bash session inside the running container:
+
+   ```bash
+   docker exec -it talawa-api-1 /bin/bash
+   ```
+3. Inside the container, run the following command to import sample data into the database: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)